pkg/connector/handlematrix.go

```diff
@@ -239,3 +241,15 @@ func (tc *TwitterClient) HandleMatrixViewingChat(ctx context.Context, chat *brid
 	tc.client.SetActiveConversation(conversationID)
 	return nil
 }
+
+func (tc *TwitterClient) HandleMatrixDeleteChat(ctx context.Context, chat *bridgev2.MatrixDeleteChat) error {
+	if chat.Content.DeleteForEveryone {
+		return errors.New("delete for everyone is not supported")
+	}
+	if chat.Portal != nil {
```

Comment:
This != nil check is redundant, delete events can't happen for a portal that doesn't exist (ViewingChat is just a special case where nil signals not viewing any chat)